app/src/main/java/com/example/recipeapp/viewmodels/SignUpViewModel.java

```diff
@@ -0,0 +1,78 @@
+package com.example.recipeapp.viewmodels;
+
+import android.content.Intent;
+import android.util.Log;
+import android.widget.Toast;
+
+import androidx.lifecycle.MutableLiveData;
+
+import com.example.recipeapp.LoginActivity;
+import com.example.recipeapp.Models.Parse.Preferences;
+import com.example.recipeapp.Models.Parse.Taste;
+import com.example.recipeapp.SignUpActivity;
+import com.parse.ParseException;
+import com.parse.ParseUser;
+import com.parse.SaveCallback;
+import com.parse.SignUpCallback;
+
+public class SignUpViewModel {
```

Comment:
Noob question , view model needs to be extended by some class , correct ?

Reply:
https://stackoverflow.com/questions/67059010/why-we-need-to-extend-viewmodel-for-databinding-and-livedata-to-create-l
It seems it should extend ViewModel